net-misc/wget/wget-1.19.5-r1.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit flag-o-matic python-any-r1 toolchain-funcs
+
+DESCRIPTION="Network utility to retrieve files from the WWW"
+HOMEPAGE="https://www.gnu.org/software/wget/"
+SRC_URI="mirror://gnu/wget/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="alpha amd64 arm arm64 hppa ia64 m68k ~mips ppc ppc64 s390 sh sparc x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
```

Comment:
You can not commit straight to stable. Thus, you can't remove older existing version either with this bump. 

I have a feeling this needs a bug discussion first before it gets merged, so you might want to open one. System packages are delicate...

Reply:
Does not seem so delicate ... Just the author of the commits seemed wrong ;). @Whissi

Reply:
juippis was right. You normally cannot do what you proposed. But as base-system project member I decided to go this route after reviewing patch history and running some tests.